internal/web/ui/src/features/component/ComponentView.tsx

```diff
@@ -47,6 +48,24 @@ export const ComponentView: FC<ComponentViewProps> = (props) => {
     );
   }
 
+  function liveDebuggingButton(): ReactElement | string {
+    if (useRemotecfg) {
+      return 'Live debugging is not yet available for remote components';
+    }
+
+    if (!liveDebuggingEnabled) {
+      return <></>;
```

Comment:
instead of returning nothing, we could put a message 'Live debugging is not yet available for this component'

Reply:
Yup I have made the changes